benchmark/src/main/java/com/google/samples/apps/nowinandroid/baselineprofile/BaselineProfileGenerator.kt

```diff
@@ -45,6 +46,12 @@ class BaselineProfileGenerator {
                 findObject(By.text("Interests"))
                     .click()
                 waitForIdle()
+                findObject(By.text("Accessibility")).scroll(Direction.DOWN, 2000f)
```

Comment:
One more thing, the 2000% look weird. I had troubles with `scroll` because it weirdly scrolls not enough .. I was using `fling` which more reliably scrolls the content.

Reply:
@keyboardsurfer can you please resolve this issue?